### ISSUE TITLE
[1.1.17][OSX] Rodeo ignores dot files display setting

### ISSUE THREAD

**Issue by nbara** [yhat/rodeo#183]
1. Uncheck 'Display Dot Files' in preferences. 
2. Restart Rodeo
3. Dot files still visible in explorer pane

my .rodeorc looks like this

```
{
  "id": "b465f1d0967811e58db8b500094b1990",
  "fontType": "Helvetica Neue",
  "paneVertical": "63.54443853113358%",
  "paneHorizontalRight": "47.729468599033815%",
  "paneHorizontalLeft": "66.3768115942029%",
  "pythonCmd": "/Users/nbara/anaconda/bin/python",
  "defaultWd": "/Users/nbara",
  "displayDotFiles": false,
  "trackingOn": false,
  "theme": "css/styles-presentation.css",
  "editorTheme": "ace/theme/tomorrow_night_eighties",
  "version": "1.1.17",
  "fontSize": 12
}
```

**Comment by nbara** [comment]
Still present in 1.1.19

**Comment by glamp** [comment]
fixed in 1.3.1

**Comment by nbara** [comment]
Where can I download 1.3.1 ? I'm still stuck on 1.3.0 ...

**Comment by nbara** [comment]
Actually the bug is still present in 1.4.8

**Comment by TakenPilot** [comment]
It was!  The version _right after that_ though has the fix, which is 1.4.9 and consequently [Rodeo 2.0.0](https://github.com/yhat/rodeo/releases/tag/v2.0.0).
